incubator/binding-grpc/src/main/java/io/aklivity/zilla/runtime/binding/grpc/internal/stream/GrpcServerFactory.java

```diff
@@ -0,0 +1,1369 @@
+    private final AbortFW abortRO = new AbortFW();
+    private final WindowFW windowRO = new WindowFW();
+    private final ResetFW resetRO = new ResetFW();
+    private final SignalFW signalRO = new SignalFW();
+
+    private final BeginFW.Builder beginRW = new BeginFW.Builder();
+    private final DataFW.Builder dataRW = new DataFW.Builder();
+    private final EndFW.Builder endRW = new EndFW.Builder();
+    private final AbortFW.Builder abortRW = new AbortFW.Builder();
+    private final WindowFW.Builder windowRW = new WindowFW.Builder();
+    private final ResetFW.Builder resetRW = new ResetFW.Builder();
+    private final FlushFW.Builder flushRW = new FlushFW.Builder();
+
+    private final GrpcBeginExFW grpcBeginExRO = new GrpcBeginExFW();
+    private final GrpcDataExFW grpcDataExRO = new GrpcDataExFW();
+    private final HttpBeginExFW httpBeginExRO = new HttpBeginExFW();
+    private final GrpcMessageFW grpcMessageRO = new GrpcMessageFW();
+    private final HttpBeginExFW.Builder httpBeginExRW = new HttpBeginExFW.Builder();
+    private final HttpEndExFW.Builder httpEndExRW = new HttpEndExFW.Builder();
+    private final GrpcBeginExFW.Builder grpcBeginExRW = new GrpcBeginExFW.Builder();
+    private final GrpcDataExFW.Builder grpcDataExRW = new GrpcDataExFW.Builder();
+    private final GrpcMessageFW.Builder grpcMessageRW = new GrpcMessageFW.Builder();
+
+    private final MutableDirectBuffer writeBuffer;
+    private final BufferPool bufferPool;
+    private final Signaler signaler;
+    private final BindingHandler streamFactory;
+    private final LongUnaryOperator supplyInitialId;
+    private final LongUnaryOperator supplyReplyId;
+    private final LongSupplier supplyTraceId;
+    private final Long2ObjectHashMap<GrpcBindingConfig> bindings;
+    private final int grpcTypeId;
+    private final int httpTypeId;
+
+    enum ContentType
+    {
+        GRPC
+        {
+            @Override
+            public void doNetEnd(
+                GrpcServer server,
+                long traceId,
+                long authorization)
+            {
+                server.doGrpcNetEnd(traceId, authorization);
+            }
+
+            @Override
+            public void doNetAbort(
+                GrpcServer server,
+                long traceId,
+                long authorization)
+            {
+                server.doGrpcNetAbort(traceId, authorization);
+            }
+        },
+        GRPC_WEB_PROTO
+        {
+            @Override
+            public void doNetEnd(
+                GrpcServer server,
+                long traceId,
+                long authorization)
+            {
+                server.doGrpcWebNetEnd(traceId, authorization);
+            }
+
+            @Override
+            public void doNetAbort(
+                GrpcServer server,
+                long traceId,
+                long authorization)
+            {
+                server.doGrpcWebNetAbort(traceId, authorization);
```

Comment:
Should this method on server be renamed to `doGrpcWebProtoNetAbort(...)` ?
(see discussion above for `doGrpcWebNetEnd`)